src/dataset-builder/DatasetBuilder.ts

```diff
@@ -599,7 +602,7 @@ export const DatasetBuilderContents = ({
         header: featureValueGroup.name,
         values: _.map((value) => ({ name: value }), featureValueGroup.values),
       }))
-    )(dataset.snapshotBuilderSettings?.featureValueGroups);
+    )(snapshotBuilderSettings?.featureValueGroups);
```

Comment:
`featureValueGroups` is a required field in snapshotBuilderSettings so I believe we can drop the `?`:
`snapshotBuilderSettings.featureValueGroups`

Reply:
Good point!